README.md

```diff
@@ -12,17 +12,19 @@ This extension adds language support for Swift to Visual Studio Code. It support
 
 Swift support uses [SourceKit LSP](https://github.com/apple/sourcekit-lsp) for the [language server](https://microsoft.github.io/language-server-protocol/overviews/lsp/overview/) to power code completion and [LLDB](https://github.com/vadimcn/vscode-lldb) to enable debugging.
 
-The extension is developed by members of the Swift Community and maintained by the [SSWG](https://www.swift.org/sswg/). The aim is to provide a first-class, feature complete extension to make developing Swift applications on all platforms a seamless experience.
+The aim is to provide a first-class, feature complete extension to make developing Swift applications on all platforms a seamless experience.
 
 If you experience any issues or want to propose new features please [create an issue](https://github.com/swift-server/vscode-swift/issues/new) or post on the `#vscode-swift` channel on [Slack](https://swift-server.slack.com).
 
 ## Contributing
 
-The Swift for Visual Studio Code extension is a community driven project, developed by the amazing Swift community. Any kind of contribution is appreciated, including code, tests and documentation. For more details see [CONTRIBUTING.md](CONTRIBUTING.md).
+The Swift for Visual Studio Code extension is a community driven project originally created by the [Swift Server Working Group](https://www.swift.org/sswg/) and now maintained as part of the [swiftlang organization](https://github.com/swiftlang/). Contributions are appreciated, including code, tests and documentation. For more details see [CONTRIBUTING.md](CONTRIBUTING.md).
+
+To give clarity of what is expected of our members, Swift has adopted the code of conduct defined by the Contributor Covenant. This document is used across many open source communities, and we think it articulates our values well. For more, see the [Code of Conduct](https://swift.org/code-of-conduct/).
 
 ## Installation
 
-For the extension to work, you must have Swift installed on your system. Please see the [Getting Started Guide on Swift.org](https://www.swift.org/getting-started/) for details on how to install Swift on your system. Install the extension from [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=sswg.swift-lang) and open a Swift package! You'll be prompted to install and configure the CodeLLDB extension, which you should do so.
+For the extension to work, you must have Swift installed on your system. Please see the [Getting Started Guide on Swift.org](https://www.swift.org/getting-started/) for details on how to install Swift on your system. Install the extension from [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=swiftlang.vscode-swift) and open a Swift package! You'll be prompted to install and configure the CodeLLDB extension, which you should do so.
```

Comment:
Is the name fixed? `vscode-swift` both seems redundant and would suggest it excludes other IDEs that the extension should work in (e.g. #90 )

FWIW all the other 1st class language extensions just use the language name, like [`golang.Go`](https://marketplace.visualstudio.com/items?itemName=golang.Go) and [`rust-lang.rust`](https://marketplace.visualstudio.com/items?itemName=rust-lang.rust)

Reply:
I've seen Microsoft do this eg `Microsoft.vscode-python` but yeah the `vscode-` still does seem kind of redundant.

Reply:
Not sure what the convention is, but I think it would be nice to spell it out as: `swiftlang.extension` (can be read as "Swift Language Extension"). It feels natural to me, would like to know what you guys think?

Reply:
I don't find `swiftlang.swift` ("Swift Language Swift") and `swiftlang.vscode-swift` ("Swift Language VS Code Swift") to read out well and it feels redundant

Reply:
Checking with the extension's marketplace page, it describes the extension as "Swift Language Support for Visual Studio Code". So, It feels natural for me to name the extension ID as `swiftlang.support` or `swiftlang.extension`. If not for these two, going with `swiftlang.swift` will be better than going with `swiftlang.vscode-swift`. :)

Reply:
It's `org.extension_name` so `support` or `extension` could be confusion if any other extensions were added under the org

Reply:
I'm not married to the name but I would like to try and avoid conflicts in the "swiftlang" OSS namespace. The extension ID `swiftlang.swift` might cause confusion by conflicting with the swiftlang/swift repo. 

My thinking was if the publisher+name matched the GitHub org+repo it would be more consistent and discoverable.

Reply:
@0xTim I like your point about not being specifically for VS Code; compatible forks are gaining popularity (VS Codium, Cursor). It may be worth doing a pass on the language in the documentation in a follow on PR to refine the language so it's more generic.

Reply:
@0xTim, totally get your point about other forks that support VSCode extensions. If there were a consistent generic term that was well understood, that might be an alternative. VSIX is the extension name, but it's a little overloaded with connotations from "full" Visual Studio; there's also VSX (as in, the [Open VSX repository](https://open-vsx.org)). 

That said, even in the open repository, they describe it as "extensions for VS Code-compatible editors", which suggests that no other term has yet reached the desired level of general understanding. 

The other considerations that went into proposing `swiftlang.vscode-swift` were:

1. It matches the repository name, so anyone encountering this extension already has an inkling of where to find its home.
2. It is confusion-free both in the Swift context and the Visual Studio context. A user who encounters this on their system will understand what it is and where it comes from. 
3. This name is purely used as a unique identifier. We wouldn't / shouldn't use it in user-facing documentation. 

@plemarquand Good call on making the language more generic (although we'd need to test the extension in a few other places to really feel comfortable that it doesn't have any hidden dependencies on VSCode: I don't think we test it anywhere else, do we?)

Reply:
@timsneath there aren't any formalized tests with other forks but I've used Cursor casually and the extension has worked great. 

VS Codium is basically identical to VS Code but with telemetry disabled so I've got high confidence the extension will work, but I haven't tried loading the .vsix in to it yet. Will make a note that needs to be tested there.

Reply:
> @plemarquand Good call on making the language more generic (although we'd need to test the extension in a few other places to really feel comfortable that it doesn't have any hidden dependencies on VSCode: I don't think we test it anywhere else, do we?)

I think Apple legal still have to review the terms and conditions to publish on OpenVSX. So we never really treated it as an alternative platform as we weren't publishing to it (although OpenVSX have an unofficial copy up https://open-vsx.org/extension/sswg/swift-lang). Maybe now everything is moving to swiftlang we can organise an OpenVSX version.

Reply:
Yes, once we've wrapped up the work to get the newly identified extension up on the VSCode Marketplace, we'll be working with Apple legal to move forward with the OpenVSX work.